src/parser.c

```diff
@@ -205,7 +205,7 @@ enum parse_rv parse_SECP256K1TransferOutput(struct SECP256K1TransferOutput_state
                         break;
                     case TRANSACTION_TYPE_ID_PLATFORM_EXPORT:
                         should_break = ADD_PROMPT(
-                            "P to X chain",
+                            "export from P",
```

Comment:
Below we do "P chain import" and "C chain export". Should this be in a consistent style?

Reply:
It should, good catch, fixing